region/norfair/crocomire.json

```diff
@@ -55,6 +55,22 @@
           "nodeAddress": "0x00193ea",
           "doorEnvironments": [{"physics": "air"}],
           "spawnAt": 5,
+          "jumpways": [
+            {
+              "name": "Door frame",
+              "jumpwayType": "doorFrameBelow",
+              "height": 2,
+              "requires": ["f_DefeatedCrocomire"]
```

Comment:
True, and the door unlocked requirement is implicit for using jumpways in general. So I removed the requirement for the door frame now. It still applies to the platform jumpway since the spike wall and Crocomire would be in the way.

Reply:
There's still a large jumpway here with croc alive.  And croc doesn't do anything so its safe to use.  You can add another.

Reply:
The approach I'm trying to take is to only add a jumpway if we know of an application of it that is not already covered. Especially once we start doing jumpways for horizontal transitions, I think it's going to be important not to add every single platform in the room (much less every combination of possible usable length based on enemies cleared) as that would create too much clutter. In this case, I'm not sure if the shorter croc-alive runway would have any application, so leaving it out for now